docs/advanced/experimental/compound_executors.md

```diff
@@ -0,0 +1,52 @@
+(compound-executor)=
+# Compound Executors
+
+A Compound Executor is an advanced type of Executor. 
+This combines two or more Executors into one class, in order to facilitate special logic.
+Examples of this include:
+
+- combining a Storage and a Search Indexer in one, for easier access, scaling etc.
+- combining a custom ranker and an Indexer in one, for cases where `top_k` is not achieved after a ranking process filters out results
+- having Indexers that store multiple level of granularity in one Executor, for easier lookup and custom logic
+
+If you want to develop a `CompoundExecutor`-type Executor based on one of the Executors existing in Jina Hub, you have to copy-paste the code of the classes you need into your own Executor's package.
+This is because, for now, the Executors in Jina Hub can **not** be imported as Python modules.
```

Comment:
```suggestion


- combining a Storage and a Search Indexer in one, for easier access, scaling etc.
- combining a custom ranker and an Indexer in one, for cases where `top_k` is not achieved after a ranking process filters out results
- having Indexers that store multiple levels of granularity in one Executor, for easier lookup and custom logic


````{caution}
If you want to develop a `CompoundExecutor`-type Executor based on one of the Executors existing in Jina Hub, you have to copy-paste the code of the classes you need into your own Executor's package.
This is because, for now, the Executors in Jina Hub can **not** be imported as Python modules.
````

```

Reply:
seems my "` * 4" does not work on Github IDE, the closing quote is eaten by it